feat: community docs implementation (#950)
asyncapi/website: pages/community/docs/getting-started/index.md

---
title: 'Getting started'

> description: 'The essential guide to start using AsyncAPI. Keep it as your reference for all the operations that you can perform on the API resources.'

Reviewer: ```suggestion
description: 'The essential guide to get started using AsyncAPI. Consider this your reference for all the operations that you can perform on the API resources.'
```
